shell/client/shell.html

```diff
@@ -1354,8 +1361,12 @@ <h3 id="app-stats">Apps</h3>
     {{else}}
       <div class="feature-key-button-row">
         <button class="button-primary feature-key-upload-button">Upload new feature key...</button>
+        <button class="button-secondary feature-key-delete-button">Delete feature key</button>
```

Comment:
It seems wrong to me that we have a setting which, once set, cannot be undone.

It's theoretically not a problem if the feature key strictly adds features, but I'm not sure that's the case. The feature key imposes a user limit, and it's not entirely clear whether that user limit should only apply to the organization (in which case you can deconfigure your organization to avoid the limit) or to all invited users. My current thinking leans towards the latter, in which case adding a feature key in itself could actually cripple a server.

Either way, I think users will be confused if they can't undo the change. They might wonder what will happen to their server when the key expires, etc.